### PR TITLE
Formatting Pass

### PR DESCRIPTION
**Pull request by davidkwoods**
- Ran the CoreFX CodeFormatter tool on the solution
- Exempted generated code, since it'll get out of date the next time we import from Swagger

#55403

### CODE REVIEW

File: SandboxXamarinForms/SandboxXamarinForms.UWP/App.xaml.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace SandboxXamarinForms.UWP
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    sealed partial class App : Application
+    internal sealed partial class App : Application
```
Comment (brendankowitz) [inline SandboxXamarinForms/SandboxXamarinForms.UWP/App.xaml.cs]: Is this ok?

Comment (davidkwoods) [inline SandboxXamarinForms/SandboxXamarinForms.UWP/App.xaml.cs]: Nope.

File: Microsoft.HealthVault/ServiceLocator.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿namespace Microsoft.HealthVault
+﻿
+namespace Microsoft.HealthVault
```
Comment (gcorvera) [inline Microsoft.HealthVault/ServiceLocator.cs]: Missing copyright header

File: Microsoft.HealthVault/SdkTelemetryInformation.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿namespace Microsoft.HealthVault
+﻿
+namespace Microsoft.HealthVault
```
Comment (gcorvera) [inline Microsoft.HealthVault/SdkTelemetryInformation.cs]: >icrosoft.H [](start = 11, length = 10)

Missing copyright header

File: Microsoft.HealthVault/IServiceLocator.cs [hunk 4]
```diff
@@ -1,4 +1,5 @@
-﻿namespace Microsoft.HealthVault
+﻿
+namespace Microsoft.HealthVault
```
Comment (gcorvera) [inline Microsoft.HealthVault/IServiceLocator.cs]: >Microsoft.HealthVault [](start = 10, length = 21)

Missing copyright header

**Comment by gcorvera** [comment]
> using Microsoft.HealthVault.Exceptions;

Missing copyright header

---
Refers to: Microsoft.HealthVault.Client.Android/AndroidBrowserAuthBroker.cs:6 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

**Comment by gcorvera** [comment]
> namespace Microsoft.HealthVault.Client

We should decide if we want to put the using statements inside the namespace or outside, many classes have it inside. 

---
Refers to: Microsoft.HealthVault.Client.Android/AndroidBrowserAuthBroker.cs:8 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

File: Microsoft.HealthVault.Client.Android/AndroidHealthVaultConnectionFactory.cs [hunk 5]
```diff
@@ -1,9 +1,10 @@
+
 namespace Microsoft.HealthVault.Client
```
Comment (gcorvera) [inline Microsoft.HealthVault.Client.Android/AndroidHealthVaultConnectionFactory.cs]: Copyright is missing

**Comment by gcorvera** [comment]
> using Grace.DependencyInjection;

copyright header

---
Refers to: Microsoft.HealthVault.Client.Core/CoreClientIoc.cs:2 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

File: Microsoft.HealthVault.Client.Ios/IosHealthVaultConnectionFactory.cs [hunk 6]
```diff
@@ -1,8 +1,9 @@
-﻿namespace Microsoft.HealthVault.Client
+﻿
+namespace Microsoft.HealthVault.Client
 {
```
Comment (gcorvera) [inline Microsoft.HealthVault.Client.Ios/IosHealthVaultConnectionFactory.cs]: copyright header

File: Microsoft.HealthVault.Client.Uwp/UwpHealthVaultConnectionFactory.cs [hunk 7]
```diff
@@ -1,8 +1,9 @@
-﻿namespace Microsoft.HealthVault.Client
+﻿
+namespace Microsoft.HealthVault.Client
 {
```
Comment (gcorvera) [inline Microsoft.HealthVault.Client.Uwp/UwpHealthVaultConnectionFactory.cs]: Missing copyright header

**Comment by gcorvera** [comment]
> using System.Net.Http;

Missing copyright header

---
Refers to: Microsoft.HealthVault.UnitTest/HealthVaultSodaConnectionTests.cs:4 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

**Comment by gcorvera** [comment]
> using System.Net.Http;

I won't add more of these comments, just make sure all the classes you are touching have it.

---
In reply to: [303582970](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/38#issuecomment-303582970) [](ancestors = 303582970)

---
Refers to: Microsoft.HealthVault.UnitTest/HealthVaultSodaConnectionTests.cs:4 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

**Comment by gcorvera** [comment]
:shipit:

**Comment by davidkwoods** [comment]
> using System.Net.Http;

Looks like the formatter tool has an option to ensure that all files have a copyright header, so I'll re-run it with our header.

---
In reply to: [303583023](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/38#issuecomment-303583023) [](ancestors = 303583023,303582970)

---
Refers to: Microsoft.HealthVault.UnitTest/HealthVaultSodaConnectionTests.cs:4 in 495aa95. [](commit_id = 495aa9594a438d99b34af39a38f2a3aaa01df47d, deletion_comment = False)

**Pull request merged**